### PR TITLE
[HDInsight]Support new api version 2023-04-15-preview 

### PR DESCRIPTION
**Pull request by yuc-Li**
### All SDK Contribution checklist:
Swagger PR link Azure/azure-rest-api-specs#24270

This checklist is used to make sure that common guidelines for a pull request are followed.

 - [ ] Please open PR in Draft mode if it is:
    - Work in progress or not intended to be merged.
    - Encountering multiple pipeline failures and working on fixes.
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] I have read the contribution guidelines.
- [ ] The pull request does not introduce breaking changes.
#### General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, see this page.
#### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.
#### SDK Generation Guidelines
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The *.csproj and AssemblyInfo.cs files have been updated with the new version of the SDK. Please double check nuget.org current release version.
### Additional management plane SDK specific contribution checklist:
Note: Only applies to `Microsoft.Azure.Management.[RP] or Azure.ResourceManager.[RP]`

- [x] Include updated management metadata.
- [x] Update AzureRP.props to add/remove version info to maintain up to date API versions.
#### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add new service label and/or contact assigned reviewer.

- If the check fails at the Verify Code Generation step, please ensure:

  - Do not modify any code in generated folders.
  - Do not selectively include/remove generated files in the PR.
  - Do use generate.ps1/cmd to generate this PR instead of calling autorest directly.
Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version.
**Note: We have recently updated the PSH module called by generate.ps1 to emit additional data. This would help reduce/eliminate the Code Verification check error. Please run**
following command:
  `dotnet msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules`
#### Old outstanding PR cleanup
Please note:
If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

File: sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0-beta.2 (2023-06-16)
```
Comment (aim-for-better) [inline sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md:3]: Can we remove the beta for this version? From my side, I hope after our test we can release one version without beta

Comment (yuc-Li) [reply sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md]: Sure, updated.

File: eng/mgmt/mgmtmetadata/hdinsight_resource-manager.txt [hunk 2]
```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/hdinsight/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\source\azure-sdk-for-net\sdk
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/hdinsight/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\source\azure-sdk-for-net\sdk
```
Comment (ArthurMa1978) [inline eng/mgmt/mgmtmetadata/hdinsight_resource-manager.txt:6]: Track 2 dont use this file, please revert it

**Review by ArthurMa1978 — APPROVED** [review]

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
Please fix the ci errors

File: sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0 (2023-06-20)
```
Comment (ArthurMa1978) [inline sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md:3]: Please provide a new date

Comment (yuc-Li) [reply sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md]: Update. Please help release this[ PR](https://github.com/Azure/azure-sdk-for-net/pull/37199) to release version 1.1.0, and then release this version 1.1.1.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**